packages/components/src/Banner/Banner.tsx

```diff
@@ -84,7 +84,7 @@ export const BannerIcon = ({ color }: { color: Color }) => {
 
   return (
     <div className={clsx(styles.icon, iconAssets.style)}>
-      <iconAssets.icon role="img" title={iconAssets.title} />
+      <iconAssets.icon type="informative" title={iconAssets.title} />
```

Comment:
More info in the PR description. If you think the switch from role (img|presentation) to type (informative|decorative) is more confusing or unnecessary, I'll switch it back.